### DISCUSSION TITLE
YouTube started giving me ad blocker warnings, today

### DISCUSSION THREAD

**Discussion by Michael-Lemons** [uBlockOrigin/uBlock-issues#2863] [General]
I am getting a popup warning that I am not allowed to use an ad blocker. This is the first time after months of heavy YouTube viewing with uBlock Origin enabled. So far, I have only gotten the warning while logged in and not in a "private browsing" window.

I am using Firefox for the Windows 10 desktop, version 118.0.2 (64-bit)

**Comment by iam-py-test** [comment]
Please follow the instructions here: https://github.com/uBlockOrigin/uAssets/issues/19976#issue-1927765169
If the issue persists, comment there with the appropriate information.
For the future:
- all issues related to YouTube should be reported at https://github.com/uBlockOrigin/uAssets/issues/19976
- issues with other websites should be reported at https://github.com/uBlockOrigin/uAssets
- "uBlock issues" is for issues with uBlock Origin which are unrelated to it's filterlists (rules)

Thank you.
